src/v/serde/parquet/BUILD

```diff
@@ -119,6 +119,7 @@ redpanda_cc_library(
         ":schema",
         ":value",
         "//src/v/base",
+        "//src/v/compression",
```

Comment:
Missing a corresponding CMake change?

Reply:
Yes good catch... Can't wait to be off cmake :)

Reply:
Fixed!